proto/autokitteh/sessions/v1/svc.proto

```diff
@@ -13,6 +13,9 @@ message StartRequest {
   };
 
   Session session = 1 [(buf.validate.field).required = true];
+
+  // Helper: if set, overwrites the session's inputs.
```

Comment:
if set, *merge and possibly override* the session inputs?

Reply:
done